Libplanet/Action/ActionEvaluator.cs

```diff
@@ -590,6 +592,43 @@ ActionContext CreateActionContext(IAccountStateDelta prevStates, int randomSeed)
                 .SelectMany(group => group.OrderBy(tx => tx.Nonce));
         }
 
+        [Pure]
+        private static IEnumerable<Transaction<T>> OrderTxsForEvaluationV3(
```

Comment:
We don't have `OrderTxsForEvaluationV3` test here. Could you add some tests for this method?

Reply:
Actually, `OrderTxsForEvaluation()` in `ActionEvaluatorTest.cs` covers all cases. 🙃